shared/troubleshooting/jetson-tx2-nx-devkit.md

```diff
@@ -0,0 +1,26 @@
+
+### Unable to flash a board that was previously running L4T 28.x
```

Comment:
The TX2 NX was launched recently and never had any L4T 28.X support. What we can include for it is just:

### Custom device tree support

Loading of custom device trees (DT) in balenaOS is supported only by the Jetson Nano and Jetson TX2 family of devices, which have u-boot support. The complete list of devices supporting custom DT and how to apply them is available in the [hardware section][hardware section].

[hardware section]:/learn/develop/hardware/i2c-and-spi/#jetson-devices

Reply:
Thanks for this!! Let me know what else you want to see added.

Reply:
Added this bit